### PR TITLE
[Doc]Add statement for supporting remtoe redis

### PR DESCRIPTION
**Pull request by WangTaoTheTonic**
## Why are these changes needed?
The feature has been merged. We need to add some doc.

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by WangTaoTheTonic** [comment]
@wuisawesome Hi Alex, do you have a minute to take a quick review?

File: doc/source/cluster/cloud.rst [hunk 1]
```diff
@@ -295,6 +295,10 @@ random port.
 The command will print out the address of the Redis server that was started
 (the local node IP address plus the port number you specified).
 
+.. note::
+
+    If you already has remote redis instances, you can specify `--address=ip1:port1,ip2:port2...` to use them. The first one is primary and left are shards. Ray will use them if the primary one if it is reachalbe, otherwise fallback to create on local.
```
Comment (wuisawesome) [inline doc/source/cluster/cloud.rst:300]:
```suggestion
    If you already has remote redis instances, you can specify `--address=ip1:port1,ip2:port2...` to use them. The first one is primary and rest are shards. Ray will create a redis instance if the default is unreachable.
```

**Review by wuisawesome — APPROVED** [review]
Left one nit, but thanks for this follow up!

**Comment by wuisawesome** [comment]
Doc build passes and this is doc only so merging.

**Pull request merged**